css/css-masking/parsing/clip-path-invalid.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>CSS Masking Module Level 1: parsing clip-path with invalid values</title>
+<link rel="author" title="Eric Willigers" href="mailto:ericwilligers@chromium.org">
+<link rel="help" href="https://drafts.fxtf.org/css-masking-1/#the-clip-path">
+<meta name="assert" content="clip-path supports only the grammar '<clip-source> | [ <basic-shape> || <geometry-box> ] | none'.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/parsing-testcommon.js"></script>
+</head>
+<body>
+<script>
+test_invalid_value("clip-path", "auto");
+test_invalid_value("clip-path", "path('m 12 34 h 56')");
```

Comment:
Why is this invalid? path() was added to CSS Shapes Level 2, right? Are you testing the first level of the specs only?

It probably will end up clipping the entire element because of the shape being 1D though.

Reply:
I followed the link from
https://drafts.fxtf.org/css-masking-1/#the-clip-path
to
https://drafts.csswg.org/css-shapes-2/#typedef-basic-shape
"The shape is computed based on the values of one of inset(), circle(), ellipse() or polygon()."

Now removed from this pull request.